CHANGELOG.md

```diff
@@ -6,6 +6,86 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 # \[Unreleased\]
 
+# 20230427.061325
+
```

Comment:
@abe-njama @ThetaSinner @jost-s 

this is a good place to give some general release notes for this minor release.

Reply:
Second beta release.

Reply:
@abe-njama yes those two words can certainly be part of a paragraph :smile: 

more importantly we could communicate what that actually means in terms of

* key features introduced
* known limitations
* types of bugs/features that might be introduced during the 0.2 series
* what this means for the existing 0.1 series

Reply:
A more secure version of Holochain that incorporates advanced network security protocols using WebRTC End-to-End Encryption and addresses known Holochain security vulnerabilities. This release introduces app-level blocking, for when an application wants to allow agents to block each other for reasons other than validation failures, user experience improvements that provide better network information, and improved system validation to make the network more secure and reliable.

@steveeJ

Reply:
nice, thanks. i rephrased it a little, WDYT?

---

This release features  more secure version of Holochain that incorporates advanced network security protocols using WebRTC End-to-End Encryption and addresses known Holochain security vulnerabilities.

More specifically, it introduces introduces:
* app-level blocking, for when an application wants to allow agents to block each other for reasons other than validation failures
* user experience improvements that provide better network information
* and improved system validation to make the network more secure and reliable.

Reply:
I like the rephrase. Looks better now.

Reply:
cool, incorporated into #2290